style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] As required for the project, Kindly make sure you `style.css` file  includes one `class` with styles for your header. This means that you should add a class in your `<h1 class="...">` tag in your `.html` file then use that `class` to style here instead of using your `h1` tag to style.